packages/lib-classifier/src/components/Classifier/components/QuickTalk/QuickTalk.js

```diff
@@ -52,75 +57,111 @@ function QuickTalk ({
-          <UnorderedList as='ul' flex={false}>
-            {comments.map(comment => {
-              const author = authors[comment.user_id]
-              const roles = authorRoles[comment.user_id]
+        <Box
+          direction='row'
+          flex={false}
+          justify='between'
+          pad='small'
+        >
+          <Anchor
+            a11yTitle={t('QuickTalk.aria.goToTalk')}
+            label={t('QuickTalk.headerLink')}
+            href={subject.talkURL}
+            target='_blank'
+            icon={<Chat />}
+          />
+          <Button
+            a11yTitle={t('QuickTalk.aria.closeButton')}
+            autoFocus={true}
+            icon={<Close size='small' />}
+            onClick={() => setExpand(false)}
+            plain
+          />
+        </Box>
+        <Box overflow='auto' tabIndex='0'>
+          <Box flex={false} pad={{ bottom: 'small', left: 'small', right: 'small' }}>
+            {comments.length > 0 && (
+              <UnorderedList as='ul' flex={false}>
+                {comments.map(comment => {
+                  const author = authors[comment.user_id]
+                  const roles = authorRoles[comment.user_id]
 
-              return (
-                <Comment
-                  key={`quicktalk-comment-${comment.id}`}
-                  comment={comment}
-                  author={author}
-                  roles={roles}
-                />
-              )
-            })}
-          </UnorderedList>
+                  return (
+                    <Comment
+                      key={`quicktalk-comment-${comment.id}`}
+                      comment={comment}
+                      author={author}
+                      roles={roles}
+                    />
+                  )
+                })}
+              </UnorderedList>
+            )}
+            {!comments.length && (
+              <Box
+                background={{ dark: 'dark-1', light: 'light-1' }}
+                margin={{ horizontal: 'none', 'vertical': 'xsmall' }}
+                pad='xsmall'
+              >
+                <Paragraph textAlign='center'>
+                  {t('QuickTalk.subjectHasNoComments')}
+                </Paragraph>
+              </Box>
+            )}
+          </Box>
+        </Box>
+        <Box
+          flex={false}
+          pad={{ horizontal: 'small', vertical: 'xsmall' }}
+        >
           {userId && (
             <PostForm
               postComment={postComment}
               postCommentStatus={postCommentStatus}
               postCommentStatusMessage={postCommentStatusMessage}
             />
           )}
+          {!userId && (
+            <Box
+              background={{ dark: 'dark-1', light: 'light-1' }}
+              margin={{ horizontal: 'none', 'vertical': 'xsmall' }}
+              pad='xsmall'
+            >
+              <Paragraph textAlign='center'>
+                {t('QuickTalk.loginToPost')}
```

Comment:
When I first saw this, I clicked on the word 'login' in order to login. A link to the login form might be useful, but I realise that we can't add NextJS links to the classifier library.

Reply:
The separation of user auth from app-project and lib-classifier is haunting me yet again 😞

Reply:
Yeah. This could be a button that fires a callback but it’s fine as it is. 

NB. In the project app, I think you can just append `?login=true` to the current page location. Obviously that won’t work in the dev classifier.